docs/documentation/head/connect.md

```diff
@@ -343,7 +356,9 @@ Connection conn = DriverManager.getConnection(url);
 
 * **disableColumnSanitiser** = boolean
 
-	Enable optimization that disables column name sanitiser.
+	Enable optimization that disables column name sanitiser. 
```

Comment:
I'm afraid this description does not make the meaning clean.

Should it be something around the following?
> `caseSensitiveColumnAccessors` (former name `disableColumnSanitiser`) with values `true`/`false`. Default `false`. When enabled, `ResultSet#get..(String)` would access column names in a case-sensitive way.